pkgs/applications/audio/mpg123/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv
 , fetchurl, alsaLib
-, buildPlatform, hostPlatform
+, hostPlatform
```

Comment:
@Ericson2314 ```Tools like LLVM that don't need up-front target platforms can safely ignore it like normal packages, and it will do no harm.```. It can safely ignore it and it won't do harm? That's overly safe! Please, remove the ", and it will do no harm" part.